public/resources/remix/index.js

```diff
@@ -6,122 +6,126 @@
       return;
     }
 
     var isTouchDevice = 'ontouchstart' in document.documentElement;
-    var detailsBar = $(".details-bar");
-    detailsBar.attr("style", "");
+    var detailsBar = document.querySelector(".details-bar");
+    
+    if (detailsBar) {
+      detailsBar.setAttribute("style", "");
+      if (isTouchDevice) {
+        detailsBar.classList.add("touch-mode");
+      } else {
+        detailsBar.classList.add("mouse-mode");
+      }
 
-    if(isTouchDevice) {
-      detailsBar.addClass("touch-mode");
-    } else {
-      detailsBar.addClass("mouse-mode");
+      detailsBar.addEventListener("click", function(event) {
+        if (event.target.classList.contains("thimble-button")) {
+          detailsBar.classList.remove("collapsed");
+          event.stopPropagation();
+        }
+      });
+
+      detailsBar.addEventListener("click", function(event) {
+        if (event.target.classList.contains("close-details-bar")) {
+          detailsBar.classList.add("collapsed");
+          event.stopPropagation();
+        }
+      });
+
+      document.querySelector(".mouse-mode").addEventListener("mouseenter", function(){
+        detailsBar.classList.remove("collapsed");
+      });
+
+      document.querySelector(".mouse-mode").addEventListener("mouseleave", function(){
+        detailsBar.classList.add("collapsed");
+      });
     }
-
-    detailsBar.on("click", ".thimble-button",function(){
-      detailsBar.removeClass("collapsed");
-      return false;
-    });
-
-    detailsBar.on("click", ".close-details-bar", function(){
-      detailsBar.addClass("collapsed");
-      return false;
-    });
-
-    $(".mouse-mode").on("mouseenter",function(){
-      detailsBar.removeClass("collapsed");
-    });
-
-    $(".mouse-mode").on("mouseleave",function(){
-      detailsBar.addClass("collapsed");
-    });
   }
 
-  function injectDetailsBar($, metadata, callback) {
-    $.ajax({
-      url: metadata.host + "/projects/remix-bar",
-      cache: false,
-      accepts: "text/html",
-      dataType: "html",
-      data: {
-        host: metadata.host,
-        id: metadata.projectId,
-        title: metadata.projectTitle,
-        author: metadata.projectAuthor,
-        updated: metadata.dateUpdated
+  function injectDetailsBar(metadata, callback) {
+    var xmlhttp = new XMLHttpRequest();
+    xmlhttp.responseType = "text";
+    var url = metadata.host + "/projects/remix-bar";
+    var data = {
+      host: metadata.host,
+      id: metadata.projectId,
+      title: metadata.projectTitle,
+      author: metadata.projectAuthor,
+      updated: metadata.dateUpdated
+    };
+
+    xmlhttp.onreadystatechange = function() {
+      if (xmlhttp.readyState === XMLHttpRequest.DONE ) {
+        var response = xmlhttp.response;
+        var textStatus = xmlhttp.statusText;
+        if (xmlhttp.status === 200) {
+          document.body.insertBefore(response, document.body.firstChild);
```

Comment:
that is not what `$(response).prependTo("body")` does - the prependTo call is actually hack that prepends the `entire body` to the element given. What you actually need to do here is
```
var fragmentBody = document.createElement("body");
var docFragment = document.createDocumentFragment();
docFragment.appendChild(fragmentBody);
fragmentBody.innerHTML = response;
var remixBar = fragmentBody.querySelector("div");
document.body.appendChild(remixBar);
callback(null);
```